packages/@aws-cdk/aws-servicecatalog/lib/product-stack-history.ts

```diff
@@ -0,0 +1,78 @@
+import { Construct } from 'constructs';
+import { CloudFormationTemplate } from './cloudformation-template';
+import { CloudFormationProductVersion } from './product';
+import { ProductStack } from './product-stack';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct as CoreConstruct } from '@aws-cdk/core';
+
+/**
+ * Properties for a ProductStackHistory.
+ */
+export interface ProductStackHistoryProps {
+  /**
+   * The base Product Stack.
```

Comment:
"The base" ?

What does "base" mean? Please talk in terms that customer using your API will care and think about. Something like:

```
The ProductStack whose history will be snapshotted to files.
```

Or something to that effect.